README.md

```diff
@@ -24,16 +24,13 @@ hypothetical `https://github.com/acme/weather.git` repository.
 on: pull_request
 steps:
   - uses: actions/checkout@v2
-  - uses: bufbuild/buf-setup-action@v0.1.0
-    with:
-      version: '0.41.0'
-  - uses: bufbuild/buf-breaking-action@v0.2.0
+  - uses: bufbuild/buf-setup-action@v0.3.0
+  - uses: bufbuild/buf-breaking-action@v0.3.0
```

Comment:
Were the `BUF_INPUT_HTTPS_*` variables still required here? Can't these be grabbed from the env?

Reply:
That change wasn't included in this release. I think that will follow.